requirements.txt

```diff
@@ -1,3 +1,4 @@
 pypif==2.0.0
 citrination_client==4.1.0
 toolz==0.8.2
+mendeleev==0.4.4
```

Comment:
mendeleev looks great, but its [pretty heavy](https://bitbucket.org/lukaszmentel/mendeleev/src/6c12a5c74948dd068d5c0b5f1a6b89bad6b3cd82/mendeleev_requirements.txt?at=default&fileviewer=file-view-default).  I see you're only using it for atomic weights; any chance we could get that info from a lighter weight dep?